### PR TITLE
Add shortcut link badges in README

### PR DESCRIPTION
**Pull request by tomasz-tomczyk**
CleanShot 2022-07-11 at 09 14 52@2x

### CODE REVIEW

**Review by vereis — APPROVED** [review]
Do you know if there are badges for the code cov?

Otherwise this looks awesome, thanks!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Sibyl
 
+[![hex.pm](https://img.shields.io/hexpm/v/sibyl.svg)](https://hex.pm/packages/sibyl)
+[![hexdocs.pm](https://img.shields.io/badge/hex-docs-lightgreen.svg)](https://hexdocs.pm/sibyl/)
+[![hex.pm](https://img.shields.io/hexpm/dt/sibyl.svg)](https://hex.pm/packages/sibyl)
+[![hex.pm](https://img.shields.io/hexpm/l/sibyl.svg)](https://hex.pm/packages/sibyl)
+
 Sibyl is a library which augments the BEAM's default tracing capabilities by hooking
 into `:telemetry`, `:dbg` (the BEAM's built in tracing and debugging functionality),
 and `OpenTelemetry`.
```
Comment (vereis) [inline README.md:10]: While you're nearby, can you change the `official documentation for Sibyl` to like to the main Sibyl module? Not the reference?

This was a mistake on my part.

**Comment by tomasz-tomczyk** [comment]
> Do you know if there are badges for the code cov?

Yes but we're not integrated with Codecov yet right? https://about.codecov.io/product/feature/badges/

**Comment by vereis** [comment]
> > Do you know if there are badges for the code cov?
> 
> Yes but we're not integrated with Codecov yet right? https://about.codecov.io/product/feature/badges/

Oh yeah, true -- and I think since we're enforcing 100% coverage codecov is maybe unnecessary :thinking: 

No problem then! LGTM

File: README.md (hunk 1 above)
Comment (tomasz-tomczyk) [reply README.md:10]: Sorted!

**Pull request merged**